dev-requirements.txt

```
@@ -39,7 +39,7 @@ nr.databind.core<0.0.17
 nr.interface<0.0.4
 
 mkdocs==1.1.2
-mkdocs-material==5.5.5
+mkdocs-material==5.5.6
```

These patch updates are getting annoying, so I'm just going to pin to the minor version from now on.

```suggestion
mkdocs-material==5.5
```